apps/web/app/lang/de.json

```diff
@@ -431,18 +518,10 @@
     "save": "Speichern",
     "stopEdit": "Abbrechen"
   },
-  "email": "E-Mail",
   "emailConfirmation": {
     "newsletterOptInMessage": "Ihre E-Mail-Adresse wurde bestätigt.",
     "newsletterOptInMessageError": "Beim Bestätigen Ihrer E-Mail ist ein Fehler aufgetreten."
   },
-  "emptyCart": "Ihr Warenkorb ist leer",
-  "emptyCartImgAlt": "Bild eines leeren Warenkorbs",
-  "emptyCartNotification": "Sie haben keine Artikel in Ihrem Warenkorb",
-  "emptyStateAltText": "Bild, das beschreibt, dass keine Artikel gefunden wurden",
-  "emptyStateText": "Wir haben keine Artikel mehr in dieser Kategorie.",
-  "emptyStateText2": "Haben Sie nicht gefunden, was Sie suchen?",
-  "emptyWishlist": "Ihre Wunschliste ist leer",
   "error": {
```

Comment:
Should move either below "errorMessages" or the other way around, "errorMessages" should move below "error"